bindings/php/README.md

```diff
@@ -0,0 +1,53 @@
+Build/Install PHP Bindings
+=============================
+
+These things need to be done to build and install PHP bindings usable in a
+normal fashion:
+
+* The Mega SDK needs to be built with the PHP bindings.
+
+The instructions given here are strictly only valid for Linux. The may need
+adaptation for other platforms.
+
+**Note:** we will refer to the root directory where SDK is downloaded as `<SDK>`
+
+## Prerequisites  
+```
+    Install `PHP` in your system  
+    Install `SWIG` in your system (it's required to generate PHP bindings)  
+    Install the required PHP dependendy `Symfony Console`, You can use `Composer` for that  
+```
+
+## How to build and run the project:  
+
+- Configure the project for PHP:
+
+```
+        ./autogen.sh
+        ./configure --disable-silent-rules --enable-php --disable-examples    
+```  
+
+- Build the shared libraries and packages:
+``` 
+        make
+```  
+
+**Note:** if your PHP version is later than `5.3.0`, you will need to make some adjustements in configuration file `php.ini`
+
+- To check your PHP version, you can run this command: `php -v`
+- To locate file `php.ini`, you can run this command in linux: `php -i | grep "Loaded Configuration File"`
+
+Uncomment and set this line like this:  
+```
+    enable_dl = On
+```
+
+Add this line at the end of the file  
+```
+    extension="<SDK>/bindings/php/.libs/libmegaphp.so"
+```
+
+## Run megacli with PHP  
+```
+ php <SDK>/examples/php/megacli.php
```

Comment:
I had to be in SDK/examples/php folder to execute there (plus I had to run composer require symfony/console there), in order for megacli.php to run. Also running lead to a crash in Ubuntu 18.04::
```Warning: Use of undefined constant MegaApi_LOG_LEVEL_ERROR - assumed 'MegaApi_LOG_LEVEL_ERROR' (this will throw an Error in a future version of PHP) in /home/mega/tmp/sdk/examples/php/megacli.php on line 667
PHP Fatal error:  Uncaught Error: Call to undefined function MegaApi_setLogLevel() in /home/mega/tmp/sdk/bindings/php/mega.php:4733
Stack trace:
#0 /home/mega/tmp/sdk/examples/php/megacli.php(667): MegaApi::setLogLevel('MegaApi_LOG_LEV...')
#1 {main}
  thrown in /home/mega/tmp/sdk/bindings/php/mega.php on line 4733

Fatal error: Uncaught Error: Call to undefined function MegaApi_setLogLevel() in /home/mega/tmp/sdk/bindings/php/mega.php:4733
Stack trace:
#0 /home/mega/tmp/sdk/examples/php/megacli.php(667): MegaApi::setLogLevel('MegaApi_LOG_LEV...')
#1 {main}
  thrown in /home/mega/tmp/sdk/bindings/php/mega.php on line 4733
```
 Having a line saying: tested ok in Fedora XX might be worth considering ;-)

Reply:
I missread the `later than` :-D. Adding those lines still throw me an error stating that Symfony/Console/Shell was not found (it seem to have been removed in an update of Symfony Console) :-(